department-of-veterans-affairs/caseflow
# Update POA page

Resolves #1085

### Description
Allow the user to access the Update POA page

### Acceptance Criteria

### Testing Plan
1. Go to the case details view for an appeal that does not have a POA
2. Click the Update POA button
3. Verify that you are taken the POA page
